test/dlc_tests/container_tests/bin/pytorch_tests/0001-test-adapt-BERT_pytorch-training-with-gloo-mpi-backend.patch

```diff
@@ -1,14 +1,14 @@
-From 041d5935dfdb0b3b0ee8ce38afbe528ffa29e58c Mon Sep 17 00:00:00 2001
-From: Your Name <you@example.com>
-Date: Thu, 6 Apr 2023 23:00:12 +0000
-Subject: [PATCH] test: adapt BERT_pytorch training with gloo and mpi backend
+From f97dbc04afbaccd14bf27f23c3a3b066e44e456d Mon Sep 17 00:00:00 2001
+From: Shibo Xing <shibox@amazon.com>
```

Comment:
do we need name here?

Reply:
sorry, I used my email info for git format-patch by mistake. I have another PR where I can immediately redo the format-patch with a default git config.

Reply:
email info reverted

Reply:
Does the commit where the email info change is reverted still need to be pushed?

Reply:
just pushed